scripts/buildClients.ts

```diff
@@ -34,7 +40,8 @@ async function buildClient(language: Language, gens: Generator[]): Promise<void>
       await run(`sbt --batch -Dsbt.server.forcestart=true +compile`, { cwd, language });
       break;
     case 'swift':
-      await run(`swift build -Xswiftc -suppress-warnings`, { cwd, language });
+      // make this work in the playground
+      if (!playground) await run(`swift build -Xswiftc -suppress-warnings`, { cwd, language });
```

Comment:
can we stick to this syntax pls

```suggestion
      if (!playground) {
        await run(`swift build -Xswiftc -suppress-warnings`, { cwd, language });
      }
```

Reply:
sure !

Reply:
I added the eslint rule for the scripts folder